### ISSUE TITLE
Modify secondary links

### ISSUE THREAD

**Issue by Mrcl1450** [BALKANGraph/OrgChartJS#260]
![image](https://user-images.githubusercontent.com/8568899/67942039-d5c80100-fbce-11e9-8894-d3072736fcb1.png)

Hello, I'm trying to archieve something like this where the lines just go from A to B. 

Modifying main links is fairly easy but about second links isn't the same. I modified the source a bit just to get the first and last point and return that but like we can see in the image isn't that easy and it creates some bugs.

Any chance that you can help me with this? Thank you.

**Comment by plamen-peshev** [comment]
Do you want the second link to be a straight line?

**Comment by Mrcl1450** [comment]
Yes also main links but in main I just need to change d="M{xa},{ya} {xb},{yb} {xc},{yc} L{xd},{yd}" to d="M{xa},{ya} L{xd},{yd}". Can't do this in secondary links.

**Comment by Mrcl1450** [comment]
Hey, following this and issue #278 I was able to do what I needed with a workaround,
I still have this problem because working with a large number of slinks my only option is to use straight lines like this 
![image](https://user-images.githubusercontent.com/8568899/73955015-965e2e80-48fa-11ea-89d7-7853c794af85.png)

![image](https://user-images.githubusercontent.com/8568899/73955338-197f8480-48fb-11ea-8129-f3f561b8ba2d.png)

Probably the problem is because I'm not managing actually calculations for this but only manipulating the array getting the first and last item.

**Comment by ZornitsaSerbezova** [comment]
Hi Mrcl1450,

This is not supported yet, but we have logged your request in our to do list and will create such links. 

Regards,
Zornitsa

**Comment by ZornitsaSerbezova** [comment]
Hi Mrcl1450,

Actually, we figured out that it can be done this way: 
https://jsfiddle.net/BALKANGraph/tesoqkzv/

**Comment by Mrcl1450** [comment]
Hey, thank you @ZornitsaSerbezova, definitely has a better result but I'm still struggling making this work with hidden nodes.
![image](https://user-images.githubusercontent.com/8568899/74146682-91022c00-4bf9-11ea-84c2-36bd29a46dc2.png)
![image](https://user-images.githubusercontent.com/8568899/74147158-94e27e00-4bfa-11ea-8613-eb4644016532.png)


Does slinks or clinks have support for tags? Maybe I could do it like main links, default lines in this part and straight lines in everywhere else?

**Comment by Mrcl1450** [comment]
https://jsfiddle.net/6haswL0o/

**Comment by ZornitsaSerbezova** [comment]
Hi Mrcl1450,

Could you try to explain in more detail and screenshots what exactly are you trying to achieve?
I don't understand why do you have hidden nodes and why do you have clinks to them .

**Comment by Mrcl1450** [comment]
Hi, 
Yeah that's one of the problems, probably with the code at my end? Sorry I assumed it was a bug with orgchart because it only appears with straight lines.

The other problem is the clink over nodes so I need to find a way to avoid that, that's why I was thinking about using default lines only in this part with tags.

**Comment by Mrcl1450** [comment]
![image](https://user-images.githubusercontent.com/8568899/74326767-dbfb7b00-4d82-11ea-86f5-fb3db187d5d3.png)

**Comment by Mrcl1450** [comment]
![image](https://user-images.githubusercontent.com/8568899/74327632-6e504e80-4d84-11ea-8b5e-54ed4fa65282.png)

**Comment by ZornitsaSerbezova** [comment]
Hi Marcelo,
Did you achieve what you need?

**Comment by Mrcl1450** [comment]
No, it's still a problem unless you guys have done something about it recently? I haven't updated in a while.

**Comment by ZornitsaSerbezova** [comment]
I don't know. I don't understand the main problem actually. I understand only that you'd like to have clinks under the nodes. This is still not supported.
If you are trying to achieve something else, please give an simple jsfiddle example with just several nodes and a screenshot of what exactly you need to achieve.

**Comment by ZornitsaPesheva** [comment]
Actually Clinks under the nodes is supported now:
OrgChart.RENDER_CLINKS_BEFORE_NODES = true;
I will close this issue as completed.
Feel free to reopen it or open another one.
